src/firebase-namespace.ts

```diff
@@ -287,10 +287,76 @@ const firebaseCredential = {
   },
 };
 
+export const credential = firebaseCredential;
+// const internal = new FirebaseNamespaceInternals();
+// export internal;
+export const internal = new FirebaseNamespaceInternals();
+
+export const SDK_VERSION = '<XXX_SDK_VERSION_XXX>';
+
+/**
+   * Initializes the FirebaseApp instance.
+   *
+   * @param {FirebaseAppOptions} [options] Optional options for the FirebaseApp instance.
+   *   If none present will try to initialize from the FIREBASE_CONFIG environment variable.
+   *   If the environment variable contains a string that starts with '{' it will be parsed as JSON,
+   *   otherwise it will be assumed to be pointing to a file.
+   * @param {string} [appName] Optional name of the FirebaseApp instance.
+   *
+   * @return {FirebaseApp} A new FirebaseApp instance.
+   */
+export function initializeApp(options?: FirebaseAppOptions, appName?: string): FirebaseApp {
+  return internal.initializeApp(options, appName);
+}
+
+/**
+ * Returns the FirebaseApp instance with the provided name (or the default FirebaseApp instance
+ * if no name is provided).
+ *
+ * @param {string} [appName] Optional name of the FirebaseApp instance to return.
+ * @return {FirebaseApp} The FirebaseApp instance which has the provided name.
+ */
+export function app(appName?: string): FirebaseApp {
+  return internal.app(appName);
+}
+
+/*
+ * Returns an array of all the non-deleted FirebaseApp instances.
+ *
+ * @return {Array<FirebaseApp>} An array of all the non-deleted FirebaseApp instances
+ */
+export function apps(): FirebaseApp[] {
+  return internal.apps;
+}
+
+export function cert(serviceAccountPathOrObject: string | object, httpAgent?: Agent): Credential {
```

Comment:
Any thoughts on where to move these?

Reply:
I'm not sure. Existing code today I think works with only the default import `firebase-admin`, but otherwise maybe auth? Or it can be kept in another file that is barrel exported so that it's still added as part of the `firebase-admin` import at the root. 

An anti-pattern but might also work is to keep it in the `auth` folder under a file and then barrel exporting it from `./index.ts` (i.e., `export * from ./auth/token-management.ts`)